charts/akeyless-api-gateway/values.yaml

```diff
@@ -149,6 +149,33 @@ akeylessUserAuth:
 #  - customer-fragments
 existingSecret: 
 
+
+# Specifies an initial configuration for the defaults section
+defaultsConf:
+  # defaultSamlAccessId:
+  # defaultOidcAccessId:
+  # defaultEncryptionKey:
+  # defaultSecretLocation:
+
+
+# Specifies an initial configuration for the caching section
+cachingConf:
+  # cacheEnable: <true/false>
+  # cacheTTL:
+  # proActiveCacheEnable:  <true/false>
+  # proActiveCacheMinimumFetchingTime:
+  # proActiveCacheDumpInterval:
+
+
+# Specifies an initial configuration for logand (log forwarding)
+# for example:
+# 
+# logandConf: |-
+#   target_syslog_host="127.0.0.1:514"
```

Comment:
enable="true"

Reply:
add link to doc:
 https://docs.akeyless.io/docs/ssh-log-forwarding#syslog-configuration

Reply:
Done